bin/pycbc_hdf_splitinj

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+import argparse
+from pycbc.inject import InjectionSet
+import h5py
+import numpy as np
+import pycbc.version
+
+"""
+Splits injections created using pycbc_create_injections into smaller sets
+"""
```

Comment:
Move the docstring between the shebang and the imports.

Reply:
After seeing the comment towards the end of the script, I would also add a sentence to the docstring explaining that the split injections are organized so as to maximize the time between injections in the output files. Otherwise, the user may simply assume that the first output file will get the first N injections in the input file, etc.